huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go

```diff
@@ -101,6 +101,13 @@ func ResourcePolicy() *schema.Resource {
 				Optional:    true,
 				Description: "The name of the replication destination region.",
 			},
+			"enable_acceleration": {
```

Comment:
this parameter not contained in buildPolicyOpDefinition

Reply:
fixed, the problem has been occured in the latest rebase option